dijkstra/Workspace_Map.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+import pygame as pg
+import cv2
+
+#pg.init()
+class drawMap:
+    def __init__(self,radius,clearance):
+        
+        self.R = radius
+        self.C = clearance
+        self.height = 200
+        self.width = 300
+        #self.Surface = pg.display.set_mode((self.width,self.height))
+        self.ax = self.R+self.C
+    
+    def getMap(self,window):
+        blue = (0,0,255)
+        white = (255,255,255)
+        window.fill(white)
+        coord_Poly = [(20-self.ax,self.height-(120-self.ax)),
+                      (25-self.ax,self.height-(185+self.ax)),
+                      (75+self.ax,self.height-(185+self.ax)),
+                      (100+self.ax,self.height-(150+self.ax)),
+                      (75+self.ax,self.height-(120-self.ax)),
+                      (50,self.height-(150-self.ax))]
+        coord_rect = [(30-self.ax,self.height-(67.5+self.ax)),
+                      (35+self.ax,self.height-(76+self.ax)),
+                      (100+self.ax,self.height-(38.6-self.ax)),
+                      (95-self.ax,self.height-(30-self.ax))]
+        #circle = [(25+self.ax,225)]
+        coord_rhom = [(225,self.height-(40+self.ax)),
+                      (250+self.ax,self.height-25),
+                      (225,self.height-(10-self.ax)),
+                      (200-self.ax,self.height-25)]
+        poly1 = pg.draw.polygon(window,blue,coord_Poly,0)
+        poly2 = pg.draw.polygon(window,blue,coord_rect,0)
+        poly3 = pg.draw.polygon(window,blue,coord_rhom,0)
+        circle = pg.draw.circle(window,blue,(225,self.height-125),25+self.ax,0)
+        ellipse = pg.draw.ellipse(window,blue,(150,self.height-100,80+self.ax,40+self.ax),0)
+        Map_ = [poly1,poly2,poly3,circle,ellipse]
+        return Map_
+        
```

Comment:
Along with making the polygon figures in the layout, I think we need to take care of the padding around the layout if we are considering the radius and clearance, padding thickness being (radius + thickness) - the robot will take some space itself and its center will not be at (x_i, y_i) if (x_i, y_i) is the starting point. It will be (x_i + radius+clearance,  y_i + radius+clearance)

Please check

Reply:
Done. 'self.shift'

Reply:
No, this is not what I meant. we need to have a boundary padding around the layout. 
refer to this image:
![image](https://user-images.githubusercontent.com/32901101/75705293-158c1b80-5c89-11ea-86bb-2fe4465420a9.png)